docs/news/2020-09-18.html

```diff
@@ -141,15 +149,15 @@ <h5 class="post-date">Fri, 18 Sep 2020</h5>
 <blockquote>
 <!--
 \setmainfont[Ligatures=TeX]TeX Gyre Pagella \setmathfontTeX Gyre Pagella Math
-
+ 
                                                      __ n=3                  n-k k
-                                          B(t)     = ❯     \ P  \binomnk(1-t)   t
-                                              cubic  ‾‾ k=0
+                                          B(t)     = ❯     \ P  \binomnk(1-t)   t                
```

Comment:
Ah, please remember to set your code editor to trim trailing whitespace. Or fix these by hand, but that's probably way more work.

Reply:
I believe these files are autogenerated by `npm start`. Do I have to check all of these `html` files manually? I use VS Code and have `"files.trimTrailingWhitespace": true` in my settings file so I can Ctrl+S them away anyway.